scripts/atlas/README.md

```diff
@@ -3,31 +3,37 @@
 The [ESM Metagenomic Atlas](https://esmatlas.com) is a repository of over database of more than 600 million metagenomic protein structures predicted by ESMFold.
 See our [blog post](https://ai.facebook.com/blog/protein-folding-esmfold-metagenomics/) to learn more.
 
+The first `v0` version of the Atlas was released on November 1st 2022,
+corresponding to the sequences in the `2022_05` release of the MGnify protein database described [here](https://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/2022_05/README.txt).
+An update `v2023_02` was released on March 17th 2023, corresponding to the sequences in the `2023_02` release of the MGnify protein database described [here](https://ftp.ebi.ac.uk/pub/databases/metagenomics/peptide_database/2023_02/).
 
-Bulk download instructions are available here, as well as foldseek databases available for download.
+Here we provide bulk download instructions for the tarballs of predicted structures, as well as foldseek databases, and LM embeddings for every sequence in the Atlas.
 
-All structures in the ESM Metagenomic Atlas were predicted with the ESMFold model released as `esm.pretrained.esmfold_v0()`.
+The structures in the ESM Metagenomic Atlas were predicted with `esm.pretrained.esmfold_v0()` for Atlas `v0`, while Atlas `v2023_02` used `esm.pretrained.esmfold_v1()`.
 We find that protein structures with predicted LDDT > 0.7 and predict TM > 0.7 to be both reasonably well structured and interesting.
-Therefore, we provide both the small set of "high quality" metagenomic structures, as well as the full set.
-The small set of structures is built from taking a 30% sequence identity clustering of MGnify90, and using the best structure from each cluster.
+Therefore, we provide both the small set of "high confidence" metagenomic structures from Atlas `v0`, as well as the full set.
+The small set of structures is built from taking a 30% sequence identity clustering of MGnify90 `2022_05`, and using the best structure from each cluster.
 The best structure is selected using the score pTM * pLDDT.
 
-The high quality structures are around 1TB in size.
+The high confidence structures are around 1TB in size.
 
-The full database is available as PDB structures and is 15TB in size.
+The full database is available as PDB structures and is 15TB in size for `v0`.
 
-We also provide a metadata dataframe: <https://dl.fbaipublicfiles.com/esmatlas/v0/stats.parquet>.
-You can load the file with pandas: `df = pd.read_parquet('stats.parquet')`.
-The dataframe has length `617051007`, the file size is 6.0GB and has md5 hash `3948a44562b6bd4c184167465eec17de`.
-This dataframe has 4 columns:
+We also provide a metadata dataframe: <https://dl.fbaipublicfiles.com/esmatlas/v2023_02/metadata-rc1.parquet>.
+You can load the file with pandas: `df = pd.read_parquet('metadata.parquet')`.
+The dataframe has length 773,846,840 records, and the file size is around 25GB.
```

Comment:
records # confirmed. I am not sure about the file size though, as I will be adding more metrics in. but not a big issue here